guides/messaging-high-availability-openshift.adoc

```diff
@@ -0,0 +1,254 @@
+...
+----
+
+== Build WildFly with messaging application image using WildFly Maven Plugin
+
+We're going to use the https://github.com/wildfly/quickstart/tree/main/remote-helloworld-mdb[WildFly Quickstart: remote-helloworld-mdb]
+to demonstrate how to build a trimmed WildFly server with a deployed messaging application. This quickstart leverages the WildFly Maven plugin
+to create a trimmed WildFly server and deploy the `remote-helloworld-mdb.war` application. It then produces a new container image based on
+the quay.io/wildfly/wildfly-runtime:latest[WildFly Runtime Image], incorporating the application.
+
+This quickstart includes a `HelloWorldMDBServletClient` servlet, which sends messages to the `HELLOWORLDMDBQueue` queue,
+and a `HelloWorldQueueMDB` message-driven bean (MDB) that consumes messages from this queue.
+
+To build the https://github.com/wildfly/quickstart/tree/main/remote-helloworld-mdb[remote-helloworld-mdb] quickstart,
+execute the following commands:
+[source, bash, options="nowrap"]
+----
+git clone git@github.com:wildfly/quickstart.git
+cd quickstart/remote-helloworld-mdb
+mvn clean package wildfly:image -Popenshift
+----
+
+There is used `openshift` profile which provides additional configuration for WildFly server to work correctly on Openshift
+environment. Maven goal `wildfly:image` instructs WildFly Maven Plugin to build a container image.
+
+You can verify newly built image by running:
+[source, bash, options="nowrap"]
+----
+$ podman images
+REPOSITORY                                                                                                                TAG             IMAGE ID      CREATED         SIZE
+localhost/remote-helloworld-mdb                                                                                           latest          cf9a174a5311  14 minutes ago  621 MB
+...
+----
+
+=== Push WildFly image to OpenShift registry
+
+We need to expose created image in the registry, so it can be later referenced from Custom Resource for WildFly operator.
+We can utilize integrated OpenShift registry and push the image into it through the ImageStream. First crate new
+ImageStream in your namespace:
+[source, bash, options="nowrap"]
+----
+oc create imagestream remote-helloworld-mdb
+----
+
+Now push your image into the ImageStream:
+
+[source, bash, options="nowrap"]
+----
+export REGISTRY="$(oc get routes -n openshift-image-registry default-route -o=jsonpath='{.spec.host}')"
+podman login --tls-verify=false -u admin  -p $(oc whoami -t) $REGISTRY
+podman tag localhost/remote-helloworld-mdb  $REGISTRY/$(oc config view --minify -o jsonpath='{..namespace}'
+)/remote-helloworld-mdb
+podman push --tls-verify=false $REGISTRY/$(oc config view --minify -o jsonpath='{..namespace}'
+)/remote-helloworld-mdb
+----
+
+=== Install WildFly operator
+
+First we need to install WildFly operator to OpenShift cluster. Because WildFly operator is not part of OpenShift OperatorHub
```

Comment:
s/install/install the/g
s/Because/Because the/g
s/WildFly operator/WildFly Operator/g